src/parser/review-incentivizer-module.ts

```
@@ -0,0 +1,174 @@
+import { Value } from "@sinclair/typebox/value";
+import {
+  ReviewIncentivizerConfiguration,
+  reviewIncentivizerConfigurationType,
+} from "../configuration/review-incentivizer-config";
+import { IssueActivity } from "../issue-activity";
+import { BaseModule } from "../types/module";
+import { Result, ReviewScore } from "../types/results";
+import { ContextPlugin } from "../types/plugin-input";
+import { GitHubPullRequestReviewState } from "../github-types";
+import { parsePriorityLabel } from "../helpers/github";
+import { getExcludedFiles } from "../helpers/excluded-files";
+import { minimatch } from "minimatch";
+
+interface CommitDiff {
+  [fileName: string]: {
+    addition: number;
+    deletion: number;
+  };
+}
+
+export class ReviewIncentivizerModule extends BaseModule {
+  private readonly _configuration: ReviewIncentivizerConfiguration | null =
+    this.context.config.incentives.reviewIncentivizer;
+  private readonly _baseRate: number;
+  private readonly _conclusiveReviewCredit: number;
+
+  constructor(context: ContextPlugin) {
+    super(context);
+    this._baseRate = this._configuration?.baseRate ?? 100;
```

This should throw an error instead of assuming 100. This value is required for the program to start.

---

Technically the right hand side value would never run because `enabled()` checks if the configuration is present so the module won't run if the configuration is `null`